modules/install.js

```diff
@@ -38,14 +38,14 @@ export function install() {
       if (!cmdPromise) return null
 
       return cmdPromise
-        .then((actions) => {
```

Comment:
This does actually catch errors within the action creator, because the cmdPromise is the promise for the actions.  If an error occurs while the cmdPromise is generating the actions, the new position of catch will still log it--which is exactly what we're interested in catching/logging.  Once we have the actions and are ready to dispatch, we want to be hands off and let errors go uncaught.